frontend/src/test/java/com/google/tripmeout/serialization/GsonPlaceVisitTypeAdapterTest.java

```diff
@@ -0,0 +1,85 @@
+package com.google.tripmeout.serialization;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonParseException;
+import com.google.tripmeout.frontend.PlaceVisitModel;
+import com.google.tripmeout.frontend.serialization.GsonPlaceVisitTypeAdapter;
+import com.google.tripmeout.serialization.testdata.TestDataAccessUtil;
+import java.util.UUID;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class GsonPlaceVisitTypeAdapterTest {
```

Comment:
Looks like all of your tests cover deserialization but no serialization. You'll definitely want to add serialization cases. (You can do so by adding round-trip cases, since you'll want to make assertions on model objects rather than serialized strings.)